src/Header.jsx

```diff
@@ -371,6 +372,19 @@ export default function Header({
     const localPlayer = metaversefile.useLocalPlayer();
     localPlayer.lookAt(object.position);
   };
+
+  const _init = async (app, canvas) => {
+    app.bindPreviewCanvas(canvas);
+    
+    const defaultAvatarUrl = './avatars/citrine.vrm';

```

Comment:
It works for this example, but this isn't loading the current avatar, or tracking changes to avatars.

Reply:
Resolved